include/wx/private/markupparserattr.h

```diff
@@ -33,16 +33,32 @@ class wxMarkupParserAttrOutput : public wxMarkupParserOutput
     // A simple container of font and colours.
```

Comment:
This is not quite true any longer, i.e. it's not trivially simple now with the attributes inheritance. I think it would be useful to mention here that it stores `xxx` which are used when rendering and `effectiveXxx` which are only used for initializing the nested attributes, as otherwise it is not really obvious why do we need both of them.